packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/TimelineHeaderRow/TimelineCollapser.css

```diff
@@ -42,3 +42,7 @@ limitations under the License.
 .TimelineCollapser--btn-expand {
   transform: rotate(90deg);
 }
+
+.TimelineCollapser--btn-size {
+  font-size: 20px;
+}
```

Comment:
It looks much bigger now. Maybe the earlier one was good.

![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/ddd95519-0600-438c-8097-054a1fc0eaa4)
![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/32a469fb-a4d9-4e56-9cc0-8a9057e00a3c)